Constitution.txt

```diff
@@ -1,490 +1,264 @@
 
 Section 2
 1: The Citizens of each State shall be entitled to all Privileges and Immunities of Citizens in the several States.
 
 2: A Person charged in any State with Treason, Felony, or other Crime, who shall flee from Justice, and be found in another State, shall on Demand of the executive Authority of the State from which he fled, be delivered up, to be removed to the State having Jurisdiction of the Crime.
 
-3: No Person held to Service or Labour in one State, under the Laws thereof, escaping into another, shall, in Consequence of any Law or Regulation therein, be discharged from such Service or Labour, but shall be delivered up on Claim of the Party to whom such Service or Labour may be due.11
-
 Section 3
-1: New States may be admitted by the Congress into this Union; but no new State shall be formed or erected within the Jurisdiction of any other State; nor any State be formed by the Junction of two or more States, or Parts of States, without the Consent of the Legislatures of the States concerned as well as of the Congress.
+1: New States may be admitted by the Congress into this Union; but no territory of any State shall be transferred to another State, or be formed into a new State as a member of the Union, unless by three-fourths assent of the people of said territory; and if joining a previously existing State, by the assent of the Government of that State according to rules laid out in the State's Constitution, or if forming a new State, with such a State Constitution as may satisfy the requirements laid upon all States by this Constitution; and no new State shall be formed from the territories of existing States which shall have a population of less than thirty thousand Citizens. When such a State is newly formed, the Citizens of that state shall be treated as having Citizenship of these United States for such term as they have been Citizens of their previous State, or of the newly admitted State.
 
-2: The Congress shall have Power to dispose of and make all needful Rules and Regulations respecting the Territory or other Property belonging to the United States; and nothing in this Constitution shall be so construed as to Prejudice any Claims of the United States, or of any particular State.
+2: The Congress shall have Power to dispose of and make all needful Rules and Regulations respecting the Territory or other Property belonging to these United States; and nothing in this Constitution shall be so construed as to Prejudice any Claims of these United States, or of any particular State; but Citizen Residents of such Territory or Property shall have the privilege to form a new State, or to join an existing State, according to the rules governing the formation or joining of a State by Citizen Residents of a territory owned by a different State.
```

Comment:
I lay it out explicitly here, though it could be inferred, that anyone populating a Federally-administered area can become a State. This would apply currently to the people of Guam, American Somoa, the US Virgin Islands, and Puerto Rico, and would also apply in the case that the US Federal Government makes claims upon and populates currently unoccupied regions like Antarctica, as well as regions of other celestial bodies.